openvino/cc/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# ******************************************************************************
+# Copyright 2017-2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ******************************************************************************
+
+set(TARGET_NAME cc)
+
+add_library(${TARGET_NAME} INTERFACE)
+
+add_library(openvino::cc ALIAS ${TARGET_NAME})
+
+target_link_libraries(${TARGET_NAME} INTERFACE openvino::itt)
+
+target_include_directories(${TARGET_NAME} INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}/include)
+
+if(${ENABLE_SUBSET} STREQUAL "ANALYZE")
```

Comment:
inside `if` we can use simple `ENABLE_SUBSET`, cmake will check whether it's a variable and expand it.

Reply:
Here, the ENABLE_SUBSET value can be ON, OFF, or ANALYZE. So we should compare the value with the string "ANALYZE".

Reply:
I understand, but cmake can automatically understand that `ENABLE_SUBSET` is a variable and take its value.

Reply:
Ok